scout/server/blueprints/variant/verification_controllers.py

```diff
@@ -72,8 +72,8 @@ def variant_verification(
     display_name = variant_obj.get("display_name")
     chromosome = variant_obj["chromosome"]
     end_chrom = variant_obj.get("end_chrom", chromosome)
-    breakpoint_1 = ":".join([chromosome, str(variant_obj["position"])])
-    breakpoint_2 = ":".join([end_chrom, str(variant_obj.get("end"))])
+    breakpoint_1 = ":".join([chromosome, str(variant_obj["position"])]) if category == "sv" else "-"
```

Comment:
:+1:
What about cancer-sv?

Reply:
Thanks! This looks straightforward enough that I could just approve the code, but it might be easier if someone on site ran the test? Otw I'll make the cg-production slack confused. 😊

Reply:
I can test it now! @karlnyr @henningonsbring @emmser @b4ckm4n you're about to get a variant verification email by me. Please ignore it, we're testing something. Thanks!